app/models/organization.js

```diff
@@ -126,6 +126,19 @@ export default class OrganizationModel extends AgentModel {
     });
   }
 
+  get kboNumber() {
+    const identifiers = this.identifiers.content;
+
+    for (const identifier of identifiers) {
+      const structuredIdentifier = identifier.get('structuredIdentifier');
+      if (identifier.idName === 'KBO nummer') {
+        return structuredIdentifier.get('localId');
+      }
+    }
+
+    return null;
+  }
```

Comment:
`relationshipName.content` is a private property so I wouldn't use that. Did you find that documented somewhere?

If you want sync access to the loaded data of async relationships you can use the reference APIs:
- https://api.emberjs.com/ember-data/release/classes/HasManyReference
- https://api.emberjs.com/ember-data/release/classes/BelongsToReference

Which would look like this:

```suggestion
  get kboNumber() {
    const identifiers = this.hasMany('identifiers').value();

    for (const identifier of identifiers) {
      const structuredIdentifier = identifier.belongsTo('structuredIdentifier').value();
      if (identifier.idName === 'KBO nummer') {
        return structuredIdentifier.localId;
      }
    }

    return null;
  }
```
It also gets rid of the `.get` which is almost always a code smell :smile:

Reply:
Ahh I see, that's much better indeed. Thanks for the feedback